makefile.shade

```diff
@@ -421,9 +426,10 @@ functions
             var nonDefaultRepos = new[]
             {
               "KRuntime",
-              "MusicStore"
+              "MusicStore",
+              "Coherence"
```

Comment:
Coherence-signed doesn't have Nuget.config so I might need to special case it. I'll add dnvm